app/models/unit.rb

```diff
@@ -12,8 +12,8 @@ class Unit < ApplicationRecord
   # relations
   has_ancestry
   belongs_to :district
-  belongs_to :unit_status
-  belongs_to :unit_type
+  belongs_to :unit_status, optional: true
```

Comment:
`unit_status` ve `unit_type` alanlarını bence boş geçmeyelim. Bu komisyon ve kurullar için `unit_status` aktif olur. `unit_type` ise yoksis referans tablosuna manuel olarak kurul, komisyon vb tipler eklenerek kullanılır.

Reply:
Kaysisin müdürlük, daire başkanlığı, kurul/komisyon vs. olarak statü alanı var aslında. Bizim referans modelimiz yöksis baz alınarak oluşturulduğu için oraya ekleme yapmamıştım.

Reply:
Bence kaysisteki birim tiplerini de bu modele ekleyelim. Bu birim tiplerinin kodlarını, yoksis referens kodlarıyla çakışmayacak şekilde belirleyerek eklemeleri yapalım. Örenk olması açısından bu birimlerin kodları 2000'den başlayabilir. (0-1999 : Yoksis, (2000-3999): Detsis gibi.

Reply:
Hepsini eklememize gerek var mı? Sadece ilk yorumunda dediğin gibi `kurul/komisyon` tipini eklesek?